lib/db/mysql/schema.sql

```diff
@@ -43,3 +43,14 @@ CREATE TABLE IF NOT EXISTS tokens (
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 ) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS developers (
+  email VARCHAR(256) NOT NULL PRIMARY KEY,
```

Comment:
TODO: here I'm planning to add something like `developerId BINARY(32) NOT NULL PRIMARY KEY,` to assign every developer an id. Would that be the best way to set an id in InnoDB?